### PR TITLE
Upload npm in pipeline

### PR DESCRIPTION
**Pull request by pdenness**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,12 +6,15 @@
 /out-tsc
 
 # dependencies
-/node_modules
+node_modules/
+projects/annotation-ui-lib/node_modules/
+**/**/node_modules/
```
Comment (velimd) [inline .gitignore:11]: question: why is this double -> **/ **/ , would **/node_modules/ work

File: README.md [hunk 2]
```diff
@@ -1,81 +1,71 @@
...
+  yarn install; export S2S_SECRET={{insert secret here}}; export IDAM_SECRET={{insert secret here}}; export APP_ENV=local; yarn start-dev-proxy;
+  ```
+- start node server
+  ```
+  yarn install; export S2S_SECRET={{insert secret here}}; export IDAM_SECRET={{insert secret here}}; export APP_ENV=local; yarn watch-dev-node;
+  ``` 
+- go to http://localhost:3000 and the viewer should load the document.
+
+### Building annotation library
+- npm run package
```
Comment (velimd) [inline README.md:8]: as mentioned before, maybe best to have this before running

File: api/index.ts [hunk 3]
```diff
@@ -18,10 +19,12 @@ process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
 router.use(serviceTokenMiddleware)
 auth(router)
 router.use(authInterceptor)
+documentsRoutes(router)
 
 if (config.configEnv !== 'prod') {
     // Uncomment to enable direct access to Microservices
     dmStoreApiRoutes(router)
+
```
Comment (velimd) [inline api/index.ts]: reason for the new line?

File: package.json [hunk 4]
```diff
@@ -35,6 +40,7 @@
     "@angular/platform-browser-dynamic": "6.1.0",
     "@angular/platform-server": "6.1.0",
     "@angular/router": "6.1.0",
+    "@hmcts/annotation-ui-lib": "1.0.7-RC",
```
Comment (velimd) [inline package.json]: this need to be updated for the new version

**Review by velimd — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (benouaer) [reply .gitignore:11]: supposed to exclude node_modules for the library, under projects/annotation-ui-lib/node_modules

**Pull request merged**